client/me/security-ssh-key/security-ssh-key.tsx

```diff
@@ -100,6 +116,36 @@ export const SecuritySSHKey = () => {
 		},
 	} );
 
+	const { updateSSHKey, isLoading: keyBeingUpdated } = useUpdateSSHKeyMutation( {
+		onMutate: () => {
+			dispatch( removeNotice( sshKeyUpdateFailureNoticeId ) );
+		},
+		onSuccess: () => {
+			dispatch( recordTracksEvent( 'calypso_security_ssh_key_update_success' ) );
```

Comment:
Can you create pull requests to record these events with Tracks?

Reply:
Here is the pr for that

Update success:
https://github.com/Automattic/tracks-events-registration/pull/1175

Update failure
https://github.com/Automattic/tracks-events-registration/pull/1176